python3.html.markdown

```diff
@@ -381,7 +379,7 @@ filled_set | other_set  # => {1, 2, 3, 4, 5, 6}
 # Let's just make a variable
 some_var = 5
 
-# Here is an if statement. Indentation is significant in python!
+# Here is an if statement. Indentation is significant in Python! Use four spaces.
```

Comment:
I'm not sure the audience understands what PEP8 is.  Elsewhere, we say, "Convention is to use lower_case_with_underscores", so how about "Convention is..."